### PR TITLE
feat/timeline refinement

### PR DESCRIPTION
**Pull request by sieuhuflit**
## Overview

- Enhance UI of event card
- Enhance UI of post card
- Update assets
- Remove wrong padding of bottom bar

## Demo

Screenshot 2023-09-07 at 14 36 14

### CODE REVIEW

File: lib/core/presentation/widgets/event/event_post_card_widget.dart [hunk 1]
```diff
@@ -49,46 +49,40 @@ class EventPostCard extends StatelessWidget {
...
-              decoration: BoxDecoration(
-                border: Border.all(
-                  color: colorScheme.outline,
-                ),
-                borderRadius: BorderRadius.circular(20),
+      child: Stack(
+        children: [
+          Container(
+            decoration: BoxDecoration(
+              border: Border.all(
```
Comment (roncoderpush) [inline lib/core/presentation/widgets/event/event_post_card_widget.dart:56]: This `Container` widget only required decoration property, consider using `DecoratedBox` for more efficient

File: lib/core/presentation/widgets/post/post_profile_card_widget.dart [hunk 2]
```diff
@@ -74,18 +75,19 @@ class PostProfileCard extends StatelessWidget {
                   Text(
                     postName,
                     style: Typo.medium.copyWith(
-                      color: colorScheme.onPrimary,
+                      color: LemonColor.white87,
                       fontWeight: FontWeight.w600,
```
Comment (roncoderpush) [inline lib/core/presentation/widgets/post/post_profile_card_widget.dart:79]: Should use `colorScheme.onPrimary.withOpacity(0.87)`

File: lib/core/presentation/widgets/post/post_profile_card_widget.dart [hunk 3]
```diff
@@ -99,7 +101,7 @@ class PostProfileCard extends StatelessWidget {
                 Text(
                   postText,
                   style: Typo.medium.copyWith(
-                    color: colorScheme.onPrimary,
+                    color: LemonColor.white87,
                     fontWeight: FontWeight.w400,
```
Comment (roncoderpush) [inline lib/core/presentation/widgets/post/post_profile_card_widget.dart:105]: Should use `colorScheme.onPrimary.withOpacity(0.87)`

**Review by roncoderpush — CHANGES_REQUESTED** [review]

File: lib/core/presentation/widgets/post/post_profile_card_widget.dart (hunk 3 above)
Comment (sieuhuflit) [reply lib/core/presentation/widgets/post/post_profile_card_widget.dart:104]: Ah I see 👍

**Review by roncoderpush — APPROVED** [review]

**Pull request merged**